.gitignore

```diff
@@ -6,4 +6,5 @@ PyDynamic.egg-info/
 .coverage
 .coverage.*
 pytest.xml
-prep_test_DFT.py
\ No newline at end of file
+prep_test_DFT.py
+.hypothesis
```

Comment:
This is a folder which hypothesis creates to store the previously failed test cases and retry them ever after. I think, it is worth to check this in as well, which we should discuss, but nevertheless it needs to be included here, because every _pytest_ execution will create this folder now.